buildh/lipids/Berger_POPC.json

```diff
@@ -0,0 +1,43 @@
+{
+  "resname": ["POPC", "PLA", "POP"],
+  "C1": ["CH3", "N4", "C5"],
```

Comment:
One other question, is it possible to add comment (with `#`) in the json file? With the former `dic_lipids.py` I found it useful to describe the different part of the lipid.

Reply:
no, comments are not allowed in json :/

Reply:
Too bad 😢

Reply:
Yes, I didn't know either until I created the json...